Source/ACE/Entity/WorldObject.cs

```diff
@@ -160,101 +160,101 @@ public virtual void SerializeCreateObject(BinaryWriter writer)
             if ((WeenieFlags & WeenieHeaderFlag.PuralName) != 0)
                 writer.WriteString16L(GameData.NamePlural);
 
-            if ((WeenieFlags & WeenieHeaderFlag.ItemCapacity) != 0)
-                writer.Write(GameData.ItemCapacity);
+            if (((WeenieFlags & WeenieHeaderFlag.ItemCapacity) != 0) && GameData.ItemCapacity != null)
```

Comment:
the way I read this (and the others belows) is if ItemCapacity is one of the flags in the WeenieFlag AND GameData.ItemCapacity is not null then write the data, is that right? If so, I believe this will lead the client to choke on the message because the WeenieFlag is sent telling it to expect ItemCapacity but then we provide it no data (or wrong data from another later write.